src-docs/src/theme_next_dark.scss

```diff
@@ -9,10 +9,7 @@
  * GitHub history for details.
  */
 
-// sass-lint:disable no-url-domains, no-url-protocols
-@import url('https://fonts.googleapis.com/css2?family=Inter:wght@300;400;500;600;700&family=Roboto+Mono:ital,wght@0,400;0,700;1,400;1,700&display=swap');
```

Comment:
Should this be replaced with Source Sans & Source Code fonts?

Reply:
Yep! i knew i forgot something. Thanks.

Reply:
To note: here, this is to enable the fonts on the docsite, not to package the fonts with OUI. OUI doesn't package any fonts out of the box, that is up to the implementor